lectures/greek_square.md

```diff
@@ -15,33 +15,31 @@ kernelspec:
 
 ## 引言
 
-{cite}`russell2004history` 第24章关于早期希腊数学和天文学的内容中包含了这段引人入胜的段落：
+{cite}`russell2004history` 第24章关于早期希腊数学和天文学的内容包含了这段引人入胜的段落：
 
  ```{epigraph} 
- 2的平方根是第一个被发现的无理数，早期的毕达哥拉斯学派就已知晓，并发明了巧妙的方法来逼近其值。
+ 早期的毕达哥拉斯学派发现了第一个无理数，即2的平方根，并发明了巧妙的方法来逼近其值。
  
- 最佳方法如下：形成两列数字，我们称之为 $a$ 列和 $b$ 列；每列都从1开始。在每一步中，下一个 $a$ 是通过加上最后一个 $a$ 和已获得的 $b$ 形成的；
- 
- 下一个 $b$ 是通过加上前一个 $a$ 的两倍和前一个 $b$ 形成的。
+ 最佳方法如下：形成两列数字，我们称之为 $a$ 列和 $b$ 列；每列都从1开始。在每一步中，下一个 $a$ 是通过加上最后一个 $a$ 和已获得的 $b$ 形成的；下一个 $b$ 是通过加上前一个 $a$ 的两倍和前一个 $b$ 形成的。
  
  这样获得的前6对数是 $(1,1), (2,3), (5,7), (12,17), (29,41), (70,99)$。
  
- 在每一对中，$2a - b$ 等于1或-1。因此，$b/a$ 接近2的平方根，并且每进行一步就会更接近。
+ 在每一对中，$2a^2 - b^2$ 等于1或-1。因此，$b/a$ 接近2的平方根，并且每进行一步就会更接近。
```

Comment:
@HumphreyYang is this a typo in the English language lecture?

Reply:
Yes, and @Yiounah will open a PR on English side!

Reply:
thanks @Yiounah